saleor/graphql/product/mutations/product_variant/product_variant_update.py

```diff
@@ -184,6 +180,14 @@ def construct_instance(cls, instance, cleaned_input):
             generate_and_set_variant_name(instance, cleaned_input.get("sku"))
         return instance
 
+    @classmethod
+    def diff_instance_data_fields(cls, fields, old_instance_data, new_instance_data):
```

Comment:
This could be a utility function somewhere in the `saleor/core` modules for future usage; it's also not mutation-specific, so it doesn't have to be part of this class.

Reply:
I moved it to our base model mutation, since in future we wanna propagate update fields across saleor mutations, seems like the place to fit.